src/app/pages/TopicPage/Pagination/index.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import clamp from 'ramda/src/clamp';
+import { number } from 'prop-types';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import { C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+import buildBlocks, { TYPE, VISIBILITY, AVAILABILITY } from './buildBlocks';
+import { Ellipsis, LeftChevron, RightChevron } from './icons';
+
+const visibilityToMediaQuery = visibility =>
+  ({
+    // TODO: minimum width being specified here as a future PR is implementing a totally different design for group 1
+    [VISIBILITY.MOBILE_ONLY]: `@media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+      display: inline-block;
+    }`,
+    // TODO: minimum width being specified here as a future PR is implementing a totally different design for group 1
+    [VISIBILITY.TABLET_DOWN]: `@media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+      display: inline-block;
+    }`,
+    [VISIBILITY.TABLET_UP]: `@media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+      display: inline-block;
+    }`,
+    [VISIBILITY.DESKTOP_ONLY]: `@media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+      display: inline-block;
+    }`,
+  }[visibility] || '');
+
+const OL = styled.ol`
+  list-style: none;
+  padding: 0;
+  text-align: center;
+`;
+
+const Block = styled.li`
+  ${({ service }) => getSansBold(service)};
+  ${({ visibility }) => visibilityToMediaQuery(visibility)}
+  display: inline-block;
+  background: #eee;
+  width: 2.5rem;
+  height: 2.5rem;
+  line-height: 2.5rem;
+  text-align: center;
+  margin-right: 0.25rem;
+  &:last-of-type {
+    margin-right: 0;
+  }
+  svg {
+    fill: currentColor;
+    width: 1rem;
+    height: 1rem;
+    position: relative;
+    top: 0.2rem;
+  }
+`;
+
+const ActiveBlock = styled(Block)`
+  color: ${C_WHITE};
+  background: #696969;
```

Comment:
I know it's annoying but I think these colours should be added to psammead styles or we should reference existing colours? As we make chameleon changes it'll be very difficult to make global changes without doing this